### PR TITLE
Add additionalPrinterColumns for Placement

### PR DESCRIPTION
**Pull request by vincent-pli**
Make `placement` more friendly to avoid cannot get the result unless going to the details.
```
NAME         SUCCEEDED   REASON
placement1   False       NotAllDecisionsScheduled
```
and 
```
NAME         SUCCEEDED   REASON
placement1   True        AllDecisionsScheduled
```

### CODE REVIEW

File: deploy/hub/placements.crd.yaml [hunk 1]
```diff
@@ -262,6 +262,13 @@ spec:
                   ManagedClusters
                 type: integer
                 format: int32
+    additionalPrinterColumns:
+    - name: Succeeded
+      type: string
+      jsonPath: ".status.conditions[?(@.type==\"PlacementSatisfied\")].status"
```
Comment (qiujian16) [inline deploy/hub/placements.crd.yaml:268]: I think you would also  add  the number of selected clusters.

**Comment by qiujian16** [comment]
this pr should be pushed to API repo at first.

File: deploy/hub/placements.crd.yaml (hunk 1 above)
Comment (vincent-pli) [reply deploy/hub/placements.crd.yaml:268]: Do you means this: @qiujian16 
```
NAME         SUCCEEDED   REASON                     SELECTEDCLUSTERS
placement1   False       NotAllDecisionsScheduled   2
```

Comment (qiujian16) [reply deploy/hub/placements.crd.yaml:268]: yes, but I think the  PR should be made in api  and sync to here.

**Comment by vincent-pli** [comment]
Made PR in repo `api`:
https://github.com/open-cluster-management-io/api/pull/89
@qiujian16

**Pull request closed without merging**